Add the primary detector info to the XRSTimeSeries if info exists in file (#6454)
sunpy/sunpy: changelog/6454.feature.rst

> Adds in abailabilyi to parse the primary detector information from the GOES-R XRS data if available to XRSTimeSeries.

Reviewer: ```suggestion
`~.XRSTimeSeries` is now able to parse the primary detector information from the GOES-R XRS data if available.
```
